pkg/kotsadm/main.go

```diff
@@ -371,14 +362,18 @@ func ensureKotsadm(deployOptions types.DeployOptions, clientset *kubernetes.Clie
 		return errors.Wrap(err, "failed to ensure kotsadm exists")
 	}
 
-	if err := ensureAPI(&deployOptions, clientset); err != nil {
-		return errors.Wrap(err, "failed to ensure api exists")
+	if err := ensureApplicationMetadata(deployOptions, clientset); err != nil {
+		return errors.Wrap(err, "failed to ensure custom branding")
 	}
 
 	if err := ensureOperator(deployOptions, clientset); err != nil {
 		return errors.Wrap(err, "failed to ensure operator")
 	}
 
+	if err := removeNodeAPI(&deployOptions, clientset); err != nil {
+		log.Error(errors.Errorf("Failed to remove deprecated API: %v", err))
```

Comment:
technically it's not deprecated, it's just removed.

Reply:
ha!  well, if on error we fail to remove it... :)